lib/ipinfo.rb

```diff
@@ -66,6 +66,20 @@ def details(ip_address = nil)
         Response.new(details)
     end
 
+    def locations(ips)
```

Comment:
Can we rename this to `get_map_url` and JSON-decode the `res.body` internal to this function, and return the `reportUrl` key from it?

This is e.g. done in the PHP SDK:

https://github.com/ipinfo/php/blob/master/src/IPinfo.php#L234-L260